src/client_side_reply.cc

```diff
@@ -1562,17 +1561,14 @@ clientReplyContext::buildReplyHeader()
         Auth::UserRequest::AddReplyAuthHeader(reply, request->auth_user_request, request, http->flags.accel, 0);
 #endif
 
-    /* Append X-Cache */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE, "%s from %s",
-                      is_hit ? "HIT" : "MISS", getMyHostname());
-
-#if USE_CACHE_DIGESTS
-    /* Append X-Cache-Lookup: -- temporary hack, to be removed @?@ @?@ */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE_LOOKUP, "%s from %s:%d",
-                      lookup_type ? lookup_type : "NONE",
-                      getMyHostname(), getMyPort());
-
-#endif
+    SBuf cacheStatus(uniqueHostname());
+    cacheStatus.append(is_hit ? ";hit" : ";fwd=miss");
```

Comment:
I suggest dropping `fwd=miss` reporting for now:

* Its current implementation will often lie or violate the Cache-Status definition for the `fwd` parameter _name_: The request may have been served by Squid without forwarding of any kind.
* Its current implementation will often lie or violate the Cache-Status definition for the `miss` parameter _value_: The request may have been forwarded even though Squid had a matching response.
* The parameter presence conveys no new information -- the lack of `hit` is sufficient -- but wastes resources.

I will come back to the `fwd=stale` change [request](https://github.com/squid-cache/squid/pull/705#discussion_r472322494) once this issue is agreed upon.

Reply:
I see. Okay that is acceptable.